src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,67 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+interface Tab {
+  id: string;
+  title: string;
+  content: string;
+}
+
+interface Props {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tabId: string) => void;
+}
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+  const handleTabSelect = (tabId: string) => {
+    if (!tabs.find(tab => tab.id === tabId)) {
+      onTabSelected(tabs[0].id);
+    }
+
+    if (tabId === selectedTabId) {
+      return;
+    }
+
+    onTabSelected(tabId);
+  };
```

Comment:
Let's use this logic to onTabSelected function

Reply:
As I understand this, I moved logic to App.tsx, and send onTabSelected already with this logic. But, I can't move second if, as I need it to prevent onTabSelected from being reinvoked if current tab is active.